src/pageServices/contentScripts/selectable.js

```diff
@@ -254,18 +269,21 @@ export const selectable = () => {
 
       // setTimeout to allow click listeners in other scripts (eg. contextmenu.js) work correctly
       setTimeout(function () {
-        if (a && a.parentNode) a.parentNode.removeChild(a);
+        removeRect();
       }, 100);
 
+      console.log("selected", selected);
+      console.log("deselected", deselected);
```

Comment:
Is it forgotten debug code or we really need it in production?

Reply:
thank you! removed